src/pkg/utils/exec/exec.go

```diff
@@ -207,3 +209,17 @@ func GetOSShell(shellPref types.ZarfComponentActionShell) (string, string) {
 func IsPowershell(shellName string) bool {
 	return shellName == "powershell" || shellName == "pwsh"
 }
+
+// ExitOnInterrupt catches an interrupt and exits with ErrorCode
+func ExitOnInterrupt(ErrorCode int, ErrorMessage string) chan os.Signal {
+	c := make(chan os.Signal, 2)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-c
+		if ErrorMessage != "" {
+			message.Warnf(ErrorMessage)
+		}
+		os.Exit(ErrorCode)
```

Comment:
```suggestion
		message.Fatal(lang.ErrInterrupt, lang.ErrInterrupt.Error())
```

This might be simpler for now and would show be more consistent with other interrupts (like when a confirmation is canceled with an interrupt).

This changes Warnf to Fatal to print a red error rather than a warning and remove formatting (since that can cause garbled text to show if the text contains formatting shortcodes).  It also changes `lang.ErrInterrupt` to be a reusable error rather than a message (see the bottom of `lang/english.go`)

![image](https://github.com/defenseunicorns/zarf/assets/3977569/8033153c-6416-427b-b20f-a5c07bcddc76)